taskcluster/ci/release/kind.yml

```diff
@@ -44,14 +44,14 @@ task-defaults:
       - . ~/.nvm/nvm.sh
       - nvm install $NODE_VERSION
       # set up yarn
-      - curl -o- -L https://yarnpkg.com/install.sh | bash
-      - export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
```

Comment:
No longer need to install and set up yarn now that we can use [corepack](https://nodejs.org/api/corepack.html).

Reply:
Running a `staging-release` here https://community-tc.services.mozilla.com/tasks/groups/AiXA8f9bS_qhQjB7nzi1NA.

Reply:
Need to fix the problem in that staging release. Switching to WIP again.